### ISSUE TITLE
Collecting Data

### ISSUE THREAD

**Issue by jolenele** [jolenele/Model-NPL-Object-Detection-Medical-Bills#4]

**Comment by jolenele** [comment]
Regular (non-medical) receipts have been collected. They have been labeled. 
1857 data points are split into: 

- Training set: 1.3k images
- Validation set: 281 images
- Testing set: 272 images

They haven't been preprocessed yet, and still waiting on the format to be decided
